### ISSUE TITLE
Prometheus instead of InfluxDB?

### ISSUE THREAD

**Issue by stoleyourburger** [vitaliy-sk/keenetic-grafana-monitoring#14]
Hello.

Please do not consider this request as a demand, but more of a question of curiosity matter.
Would it be a big work to make this monitoring also work with Prometheus instead of Influx DB? Or maybe there is some forked version of that script somewhere that is adapted for Prometheus that I don't know about?

Thanks.

**Comment by vitaliy-sk** [comment]
Hello @stoleyourburger 

Didn't plan to add Prometheus support, feel free to contribute if you really need it. 
As an alternative, you can run node_exporter directly on the router, it exports many OS metrics, but it's generic. 

I did some tests and it's working:

1. Download and unpack `node_exporter-1.5.0.linux-mipsle.tar` from https://github.com/prometheus/node_exporter/releases/tag/v1.5.0
2. Copy to router to `/opt/bin/`
3. Connect via SSH
* chmod +x /opt/bin/node_exporter
* vim /opt/etc/init.d/S99nodeexporter

```
#!/bin/sh

[ "$1" != "start" ] && exit 0

nohup /opt/bin/node_exporter >/dev/null 2>&1 &
```
* /opt/etc/init.d/S99nodeexporter start
4. Configure firewall and your Prometheus to collect metrics

**Comment by mazzz1y** [comment]
Hi guys,

I have a similar issue where I have numerous software integrated with Prometheus, and I did not want to start InfluxDB only for the keenetic exporter.

Therefore, I created a dirty solution that converts InfluxDB data to the Prometheus scrape format using [influxdb_exporter](https://github.com/prometheus/influxdb_exporter). This solution has been working wonderfully for me for over a year.

Here is my repository containing instructions and a Docker container ready to launch:
https://github.com/mazzz1y/keenetic-exporter
